db/models.py

```diff
@@ -1 +1,40 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name: str = models.CharField(max_length=255, unique=True)
+    description: str = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Skill(models.Model):
+    name: str = models.CharField(max_length=255, unique=True)
+    bonus: str = models.CharField(max_length=255)
+    race: Race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
Using type hints like 'race: Race =' for Django model fields is unconventional. Consider removing the type hints to adhere to Django's standard practices.